webpack/scenes/RedHatRepositories/components/RepositorySetRepositories.js

```diff
@@ -28,7 +28,36 @@ class RepositorySetRepositories extends Component {
     }
 
     const availableRepos = [...data.repositories.filter(({ enabled }) => !enabled)]
-      .sort((repo1, repo2) => yStream(repo1.releasever || '') - yStream(repo2.releasever || ''))
+      .sort((repo1, repo2) => {
+        const repo1YStream = yStream(repo1.releasever || '');
+        const repo2YStream = yStream(repo2.releasever || '');
+
+        if (repo1YStream < repo2YStream) {
+          return -1;
+        }
+
+        if (repo2YStream < repo1YStream) {
+          return 1;
+        }
+
+        if (repo1.arch === repo2.arch) { // arch is the same
+          // look at major, minor
```

Comment:
oh, one ask. these comments are pretty self-explanatory & think they can be removed